xcode/examples.xcodeproj/project.pbxproj

```diff
@@ -628,7 +629,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				DEVELOPMENT_TEAM = VU3TCKU48B;
+				DEVELOPMENT_TEAM = S42TTF822Y;
```

Comment:
I don't know much about Mac developement, so before merging this: Is this `DEVELOPMENT_TEAM` team safe to be part of a public repository, or is this something security related that shouldn't be publicly visible?

Reply:
I have removed the Xcode directory as it is no longer used to build the Vulkanexamples.

This has removed the above apple development team references.

When you build a application for iOS or MacOS apple require you to sign them with a a certificate generated by Xcode, if you are testing them you can use a developers certificate which is free. If you want to publish a application you need to setup a developers account and pay the annual fee.

If somebody wants to use the Vulcan examples they will need to build them on their MacBook and sign them to use them on their own MacBook only. (Unless the have a valid developers account)

Reply:
Thanks for the clarification. I recently did some renaming and had to update a mac file. If you resolve the open merge conflict I'll merge this asap.

Reply:
I am unable to resolve the conflicts as I do not have 'write access'

Reply:
That's odd. You should be able to resolve the conflict locally, by merging master (from my repo) into your branch.